docs/docs/debugging-the-build-process.md

```diff
@@ -72,6 +72,10 @@ If you use VS Code and its integrated terminal, you can configure it to automati
 
 2.  Using VS Code's integrated terminal run `node --nolazy --inspect-brk node_modules/.bin/gatsby develop` instead of `gatsby develop`
 
+> **Note:** On machines running on Windows you may get an error such as 
+> `basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')") SyntaxError: missing ) after argument list`
+> A workaround is to run `node --nolazy --inspect-brk node_modules/gatsby/dist/bin/gatsby develop`.
+
```

Comment:
Let's, instead of the note, switch command we use in Line 73 to be one you propose in this note - it's functionally the same and this way we avoid the "problem" of files in `node_modules/.bin` being different on Windows

Reply:
That's a good approach.